scripts/shared/lib/deploy_funcs

```diff
@@ -122,3 +122,17 @@ function load_deploytool() {
     . "$deploy_lib"
 }
 
+function find_submariner_namespace() {
+    kubectl get pods -A | awk '/submariner/{ print $1 }' | grep -v broker | head -n 1
```

Comment:
In my local kubectl, I see that its not recognizing the shortcut "-A". Can you change it to "--all-namespaces"?

[sgaddam@localhost submariner]$ kubectl get pods -A | awk '/submariner/{ print $1 }' | grep -v broker | head -n 1
Error: unknown shorthand flag: 'A' in -A

While we are at it, instead of querying for pods in all namespaces, do you think its better to query for daemonSets (less output to parse)
kubectl get ds --all-namespaces | awk '/submariner/{ print $1 }' | grep -v broker | head -n 1

Reply:
Better to query for pods, I'm just trying to figure out the namespace and I was trying to support both "deployments" and "daemonsets", 

since in some cases we will have just deployments.